docs/management/admin/response-actions-config.asciidoc

```diff
@@ -6,8 +6,88 @@
 :frontmatter-tags-content-type: [how-to]
 :frontmatter-tags-user-goals: [manage]
 
+Endpoint response actions involving third-party systems require additional configuration. This page explains the high-level steps you'll need to take to enable these response actions.
+
+[discrete]
+[[configure-sentinelone-response-actions]]
+== Configure SentinelOne response actions
+
+SentinelOne response actions allow you to perform bidirectional actions on protected hosts, such as directing SentinelOne to isolate a suspicious endpoint from your network, without needing to leave the {elastic-sec} UI.
+
+preview::[]
+
+.Prerequisites
 [sidebar]
 --
-[.text-center]
-**This is a placeholder for future documentation.**
+* https://www.elastic.co/pricing[Subscription level]: Enterprise 
+
+* {kibana-ref}/kibana-role-management.html#adding_kibana_privileges[{kib} feature privilege]: **Actions and Connectors : All**.
+
+* <<endpoint-management-req,{elastic-sec} feature privileges>>: **All** for the response action features, such as **Host Isolation**, that you want to perform.
+
+* Endpoints must have actively running SentinelOne agents installed.
 --
+
+Configuration requires the following general steps. Expand the steps and follow the links for detailed instructions:
+
+. **Generate API access tokens in SentinelOne.** You'll need these tokens in later steps, and they allow {elastic-sec} to collect data and perform actions in SentinelOne. 
++
+.Expand for details
+[%collapsible]
+====
+Create two API tokens in SentinelOne, and give them the least privilege required by the Elastic components that will use them:
+
+- SentinelOne integration: Permission to read SentinelOne data.
+- SentinelOne connector: Permission to perform actions on SentinelOne-protected hosts (for example, isolating and releasing an endpoint).
+
```

Comment:
This is where @patrykkopycinski 's knowledge would be good.

I would say: this Connector API key should have ability to read data (ex. we query for agent details) as well as Isolate/Release), so you may want to expand it a bit more to include something more about being able to also read data